### ISSUE TITLE
Click on grid view does not always grab scroll events

### ISSUE THREAD

**Issue by jeremypw** [elementary/photos#468]
* Click on a sidebar tag or a event to show a set of photos larger than the view.
* Hover over grid - cannot scroll or navigate with arrow keys because sidebar still has keyboard focus
* Primary click on blank part of grid - still cannot scroll but can navigate with arrow keys

This only seems to affect some categories in the sidebar (Events and Tags).  The Library and Smart Album categories are unaffected.
